### PR TITLE
Add cat photo app

### PR DESCRIPTION
**Pull request by OMOTAYO05**
## A cat photo app
** A website created using html 

>show how to add image to a webisite and how to add link to html using various element

-   `form`                       
- `label`                         
- `figcaption`  
-  `legend`  
- `footer` 
- and so on

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
 # Project Name
 
-> One paragraph statement about the project.
+> A CAT PHOTO APP BUILT USING HTML.
 
-![screenshot](./app_screenshot.png)
+![01-fcc-cat-photo-app](https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg)
```
Comment (teckcipher) [inline README.md]: The link should be your app screenshot not a photo of a cat.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,23 @@
 # Project Name
```
Comment (teckcipher) [inline README.md]: Let's add the name of the project here.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
+- HTML5 elements
+- Ordered list `<ol>`,
+  - Unordered list `<ul>`,
+  - `section`, `form`, `main`, `footer`,`figcaption`,`legend`,`label`,
+- Linters
+
+  - [Prettier](https://prettier.io/),
+  - [Stylelint](https://stylelint.io/),
+
+- Github Action
```
Comment (teckcipher) [inline README.md:20]: Can we add link for [Github Action](https://docs.github.com/en/actions)? We should also add [Vercel](https://vercel.com).

**Review by teckcipher — CHANGES_REQUESTED** [review]
There are few things left to do, you can check out Azeezat's PR to catch up with some of the suggested changes.
Aside from the below comments, your README should include the demo/deploy link of the app, the correct github repo of this app  for cloning instruction and your details in the author section.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,27 @@
-# Project Name
+# 01-FCC-CAT-PHOTO-APP
```
Comment (Gbemisola19) [inline README.md:1]: Can you remove the '01' and 'underscore'

File: README.md [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
 
-> One paragraph statement about the project.
+> A CAT PHOTO APP BUILT USING HTML.
```
Comment (Gbemisola19) [inline README.md:3]: since it is not an heading, it would look better if you  change the sentence  to a lower case

File: README.md [hunk 6]
```diff
@@ -1,20 +1,27 @@
...
 
-![screenshot](./app_screenshot.png)
+![01-fcc-cat-photo-app](app_screenshot.png)
```
Comment (Gbemisola19) [inline README.md:5]: Can we just leave it as cat photo app?

File: README.md (hunk 5 above)
Comment (Gbemisola19) [inline README.md:3]: There is no need for space between your project name and statement about the project

**Pull request merged**